.vscode/settings.json

```diff
@@ -1,4 +1,20 @@
 {
     "npm.packageManager": "yarn",
+<<<<<<< Updated upstream
     "editor.tabSize": 2
+=======
+    "editor.tabSize": 2,
+    "cSpell.words": [
+        "ciphertext",
+        "collabswarm",
+        "crdt",
+        "ipfs",
+        "libp",
+        "p",
+        "pss",
+        "tsc",
+        "websockets",
+        "wss"
+    ]
+>>>>>>> Stashed changes
```

Comment:
Looks like this merge conflict didn't get resolved properly

Reply:
Good catch!